backend/src/core/middlewares/campaign.middleware.ts

```diff
@@ -130,7 +130,7 @@ const createCampaign = async (
 const listCampaigns = async (
   req: Request,
   res: Response,
-  next: NextFunction
+  next: NextFunction // todo: remove the error handling?
```

Comment:
Yea we probably don't need `next(err)` cos if `err` thrown inside the handler that's the default behavior I think